src/components/custom/review/types.ts

```diff
@@ -2,13 +2,27 @@ import { UneditableSectionItemProps } from "@lifesg/react-design-system/uneditab
 import type { IAlertSchema, ITextSchema } from "../../elements";
 import type { IWrapperSchema } from "../../elements/wrapper";
 import type { ICustomElementJsonSchema } from "../types";
+import { BoxContainerProps } from "@lifesg/react-design-system";
 
 type TReviewSectionChildren = IAlertSchema | ITextSchema | IWrapperSchema;
 
-export interface IReviewSchema extends ICustomElementJsonSchema<"review"> {
-	label?: string | undefined;
-	description?: string | undefined;
+export type IReviewSchema = IReviewSchemaAccordion | IReviewSchemaDefault;
+export interface IReviewBase extends ICustomElementJsonSchema<"review"> {
 	items: UneditableSectionItemProps[];
 	topSection?: Record<string, TReviewSectionChildren> | undefined;
 	bottomSection?: Record<string, TReviewSectionChildren> | undefined;
 }
+export interface IReviewSchemaDefault extends IReviewBase {
+	variant?: "default" | undefined;
+	label?: string | undefined;
+	description?: string | undefined;
+}
+export interface IReviewSchemaAccordion
+	extends IReviewBase,
+		Omit<BoxContainerProps, "children" | "callToActionComponent"> {
+	variant: "accodion";
+	button?: IButtonAccordion | undefined;
+}
```

Comment:
- we always use `label` for all schemas, so you can replace `title` with `label`
- `label` should be a required key
- due to ux requirements, we need to hide some options from the accordion variant because they is not meant to be used

```
export interface IReviewBase extends ICustomElementJsonSchema<"review"> {
	label: string;
	items: UneditableSectionItemProps[];
}
export interface IReviewSchemaDefault extends IReviewBase {
	variant?: "default" | undefined;
	description?: string | undefined;
	topSection?: Record<string, TReviewSectionChildren> | undefined;
	bottomSection?: Record<string, TReviewSectionChildren> | undefined;
}
export interface IReviewSchemaAccordion
	extends IReviewBase,
		Omit<BoxContainerProps, "children" | "title" | "callToActionComponent" | "subComponentTestIds"> {
	variant: "accordion";
	button?: IButtonAccordion | undefined;
}
```